javascript/code-challenges/tree/README.md

```diff
@@ -0,0 +1,17 @@
+# Challenge Summary
+Binary Tree and BST Implementation
+
+## Challenge Description
+Accept a string with brackets and return true if the string contains balanced open and close bracket types, else return false
+
+## Approach & Efficiency
+change string to array, iterate through array.
+push open brackets into a stack, when closing bracket found, compare to top of stack.
+
+O(n) time & space
+
+## Solution
+
+- [Multi Bracket Validation](multi-bracket-validation.js) 
+
+![Whiteboard](assets/codechallenge13whiteboard.png)
```

Comment:
Looks like you are off to a start, I would suggest removing that multi bracket validation link out of here.
When you have time to come back and add code with tests, I will regrade for you!